### DISCUSSION TITLE
Display back-end error within the `bk-notifications` component.

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#372] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.4.7

### Description

Hello everyone,
I have a decorator that handles some checks during update. The decorator returns specific errors, is it possible to display these errors within the `bk-notifications` component?

For example I could receive a 400 error with this body:
```json
{"message":"Could not patch a prescription that is not pending"}
```

**Comment by clabene** [comment, marked as answer]
Hi @davidgarra.

at the moment, the `bk-notifications` component does not allow to differentiate notifications based on the content of the received error.

Possible solutions include creating a custom web component which emits different events based on the received response, specifically targeted to be listened by the `bk-notifications` component.
